### ISSUE TITLE
Predict returns a probability of 2.696539702293474e301 

### ISSUE THREAD

**Issue by joostaarts** [beniz/deepdetect#160]
Hi there, 

At times i receive a result looking like:

{"uri":"0","classes":{"last":true,"prob":2.696539702293474e301,"cat":"overig"}}}}

How should i interpret this this probability value?

**Comment by beniz** [comment]
With high certainty, there's an error in the setup of your network, or you are doing something wrong. Such values mean that the softmax layer returns infinity, and that may occur if your final layer returns zero everywhere, which indicates a mistake on your part.

If unsure and you can't debug on your own, post your exact calls to the API, server output, model description, etc...

**Comment by joostaarts** [comment]
Yes, you were right, it had to do with the creation of the network, sadly im not sure which parameter caused this though. Thanks for the pointer

**Comment by joostaarts** [comment]
Actually, it turned out to be the base_lr parameter being too high in combination with the training data
